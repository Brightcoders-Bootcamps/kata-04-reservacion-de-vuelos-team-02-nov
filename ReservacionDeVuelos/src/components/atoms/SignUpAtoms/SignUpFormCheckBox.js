import React from 'react';
import {View, Text} from 'react-native';
import {styles} from '../../../styles/SignUpStyles';
import CheckBox from '@react-native-community/checkbox';
const SignUpFormCheckBox = (props) => {
  const {checkboxMessage} = props;
  return (
    <View style={styles.checkBoxContainer}>
      <CheckBox />
      <Text>{checkboxMessage}</Text>
    </View>
  );
};

export default SignUpFormCheckBox;
