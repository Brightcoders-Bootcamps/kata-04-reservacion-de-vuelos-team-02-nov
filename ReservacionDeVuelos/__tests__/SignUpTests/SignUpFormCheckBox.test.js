import React from 'react';
import SignUpFormCheckBox from '../../src/components/atoms/SignUpAtoms/SignUpFormCheckBox';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUpFormCheckBox />).toJSON();
  expect(tree).toMatchSnapshot();
});
