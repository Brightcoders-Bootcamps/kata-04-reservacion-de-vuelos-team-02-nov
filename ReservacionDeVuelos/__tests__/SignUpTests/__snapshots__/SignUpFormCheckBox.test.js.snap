// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reders conrreclty snapshot 1`] = `ShallowWrapper {}`;
